openstates/nc/bills.py

```diff
@@ -70,10 +75,13 @@ def scrape_bill(self, chamber, session, bill_id):
             bill_type = "bill"
             bill_id = bill_id[0] + "B " + bill_id[1:]
 
-        bill_title = doc.xpath(
-            '/html/body/div/div/main/div[2]/div[contains(@class,"col-12")]/a'
-        )[0]
+        bill_title = doc.xpath("/html[1]/body[1]/div[1]/div[1]/main[1]/div[2]/div[1]")[
+            0
+        ]
         bill_title = bill_title.text_content().strip()
+        if not bill_title:
```

Comment:
is this supposed to be here? looks like debugging stuff that won't actually get called?

Reply:
Yes. It is for a few special cases where there is nothing in bill_title. It then creates a bill_title using the bill_id.

Reply:
ah, can you add a comment explaining this & drop the print statement 👍